sig-network/role-sig-chair.md

```diff
@@ -0,0 +1,70 @@
+# Role: SIG Chair
+
+## Background
+
+Please see the
+[standard description of a SIG chair](https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md#chair),
+which does a good job describing the expectations of the more “administrative”
+aspect of being a SIG chair.
+
+## What's expected
+
+Rather than repeat the doc linked above, this doc will just add some color.
+
+The minimum effective engagement for a SIG chair is to “run the show” (really
+“make sure the show is running” - delegation is good!).  That generally means
+building an agenda for and running regular meetings, curating our
+community-facing metadata, and acting as liaison to other groups (other SIGs
+and steering, mostly).
+
+This includes new-issue triage - either doing the triage or leading it (e.g. in
+the regular SIG meetings).
+
+## Above and beyond
+
+The basic expectations above should not be a huge time commitment.  SIG chairs
```

Comment:
> The basic expectations above should not be a huge time commitment.

are "basic expectations" refers to ones outlined in this doc or in the linked requirements document?  Neither should be a huge time commitment (with the common sense definition of "huge"). The sentence is, however, sounds discrediting for the work that can be done.

Reply:
If one does the minimum expectations, it really shouldn't be a ton of work.  I don't mean that dismissively or to imply there's not value to it.  It's just not going to eat 10 hours a week.  Or it shouldn't, at least.